app/src/main/java/de/danoeh/antennapod/ui/screen/playback/audio/AudioPlayerFragment.java

```diff
@@ -271,6 +271,7 @@ private void loadMediaInfo(boolean includingChapters) {
             if (media.getChapters() == null && !includingChapters) {
                 loadMediaInfo(true);
             }
+            // TODO should we load transcripts here?
```

Comment:
I don't think so. The chapters are only loaded here because we show them on the seek bar. We don't show the transcripts here, so we don't need to load them